### DISCUSSION TITLE
Permissions related to moderating users.

### DISCUSSION THREAD

**Discussion by catchlight** [craftcms/cms#15167] [Ideas]
I’ve built a portal with three users groups: manager, staff and vendor. The manager needs to access the CP, create and edit entries, and moderate users. Staff and vendor only get front end access.

I've enabled Edit users > Moderate users for the manager group, but encountered two problems:

1. Enabling this allows the manager to edit the admin account. They cannot suspend or delete the account, or change permissions, but they can edit profile fields, Addresses, and Group assignment. I feel that editing an admin account in any way should be off limits to users with lower permissions.
2. They do not seem to have the ability to activate new users — only suspend them after self-registration. What I really need is for the manager to be able to activate new users before they can access the site at all. So far as I can tell, only admins can activate new users.

**Comment by anita-chouhan** [comment]
We also need a similar feature where we can create a user group that can moderate the users of a specific group. Currently, even if the users of a moderator group can't assign users to other groups but they can still edit them.

**Reply by LarsDol** [reply]
+1!
